### PR TITLE
feature: publish blog post Nukomeet Meet-up March

### PR DESCRIPTION
**Pull request by myviridis**

### CODE REVIEW

File: content/blog/2015-04_poznan_meetup_march.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+Come and join us if you are around to talk and drink a beer in our good company.
+
+![ślepe Ryby](/assets/images/blog/SlepeRyby.jpg "SlepeRyby")
+
+### Where & When
+
+ * [Ślepe Ryby][1] at Murna 3
+ * <a href="http://www.google.com/calendar/event?action=TEMPLATE&text=Nukomeet%20Meetup&dates=20150331T190000Z/20150401T200000Z&details=Nukomeet%20POZnan%20Meetup%31March%20at%C5%9Alepe%20Ryby&location=%Murna%203%2C%20Pozna%C5%84%2C%20Polska" target="_blank"> March 31st around 19:00</a>
```
Comment (2can) [inline content/blog/2015-04_poznan_meetup_march.md]: @myviridis link url doesn't work for me, does it work for you ?

Comment (myviridis) [inline content/blog/2015-04_poznan_meetup_march.md]: No :( I must review it.

--
Kasia Popek kasia@nukomeet.com



On Thu, Mar 26, 2015, at 08:22 AM, Bartłomiej Wiaduch wrote:
> In content/blog/2015-04_poznan_meetup_march.md[1]:


> > +title: "POZnan Meetup March"
> +authors: +- kasia +tags: +- meetup +extract: Our new meet-up is
> coming soon. This time we invite you to very special place with a
> very special name. Ślepe Ryby means Blind Fish. And it's not a
> joke for 1st of April. +--- + +Come and join us if you are around
> to talk and drink a beer in our good company. + +ślepe
> Ryby + +### Where
> & When +
> + * [Ślepe Ryby][1] at Murna 3
> + * <a
>   href="http://www.google.com/calendar/event?action=TEMPLATE&text=Nukomeet%20Meetup&dates=20150331T190000Z/20150401T200000Z&details=Nukomeet%20POZnan%20Meetup%31March%20at%C5%9Alepe%20Ryby&location=%Murna%203%2C%20Pozna%C5%84%2C%20Polska"
>   target="_blank"> March 31st around 19:00</a> @myviridis[2] link url
>   doesn't work for me, does it work for you ?


> — Reply to this email directly or view it on GitHub[3].

>



Links:

  1. https://github.com/nukomeet/nukomeet.com/pull/185#discussion_r27192354
  2. https://github.com/myviridis
  3. https://github.com/nukomeet/nukomeet.com/pull/185/files#r27192354

Comment (myviridis) [inline content/blog/2015-04_poznan_meetup_march.md]: I compared your link from January30 and mine: <a
href="http://www.google.com/calendar/event?action=TEMPLATE&text=Nukomeet%20Meetup&dates=20150110T180000Z/20150110T200000Z&details=Nukomeet%20POZnan%20Meetup%20January%20at%20Cicha%20Kuna&location=%C5%9Awi%C4%99ty%20Marcin%2030%2C%20Pozna%C5%84%2C%20Polska"
target="_blank">February 10th around 19:00</a>





<a
href="http://www.google.com/calendar/event?action=TEMPLATE&text=Nukomeet%20Meetup&dates=20150331T190000Z/20150401T200000Z&details=Nukomeet%20POZnan%20Meetup%31March%20at%C5%9Alepe%20Ryby&location=%Murna%203%2C%20Pozna%C5%84%2C%20Polska"
target="_blank"> March 31st around 19:00</a>


And I found this mistake: there was % before Murna and it
shouldn't be there.

--
Kasia Popek kasia@nukomeet.com



On Thu, Mar 26, 2015, at 08:22 AM, Bartłomiej Wiaduch wrote:
> In content/blog/2015-04_poznan_meetup_march.md[1]:


> > +title: "POZnan Meetup March"
> +authors: +- kasia +tags: +- meetup +extract: Our new meet-up is
> coming soon. This time we invite you to very special place with a
> very special name. Ślepe Ryby means Blind Fish. And it's not a
> joke for 1st of April. +--- + +Come and join us if you are around
> to talk and drink a beer in our good company. + +ślepe
> Ryby + +### Where
> & When +
> + * [Ślepe Ryby][1] at Murna 3
> + * <a
>   href="http://www.google.com/calendar/event?action=TEMPLATE&text=Nukomeet%20Meetup&dates=20150331T190000Z/20150401T200000Z&details=Nukomeet%20POZnan%20Meetup%31March%20at%C5%9Alepe%20Ryby&location=%Murna%203%2C%20Pozna%C5%84%2C%20Polska"
>   target="_blank"> March 31st around 19:00</a> @myviridis[2] link url
>   doesn't work for me, does it work for you ?


> — Reply to this email directly or view it on GitHub[3].

>



Links:

  1. https://github.com/nukomeet/nukomeet.com/pull/185#discussion_r27192354
  2. https://github.com/myviridis
  3. https://github.com/nukomeet/nukomeet.com/pull/185/files#r27192354

File: content/blog/2015-04_poznan_meetup_march.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+Come and join us if you are around to talk and drink a beer in our good company.
+
+![ślepe Ryby](/assets/images/blog/SlepeRyby.jpg "SlepeRyby")
+
+### Where & When
+
+ * [Ślepe Ryby][1] at Murna 3
+ * <a href="http://www.google.com/calendar/event?action=TEMPLATE&text=Nukomeet%20Meetup&dates=20150331T190000Z/20150401T200000Z&details=Nukomeet%20POZnan%20Meetup%31March%20at%C5%9Alepe%20Ryby&location=Murna%203%2C%20Pozna%C5%84%2C%20Polska" target="_blank"> March 31st around 19:00</a>
```
Comment (2can) [inline content/blog/2015-04_poznan_meetup_march.md]: @myviridis please check dates as for me it's 21:00 - 22:00

**Pull request merged**